src/pages/api/inboundMatching.ts

```diff
@@ -89,55 +22,24 @@ export default async (req: NextApiRequest, res: NextApiResponse): Promise<void>
             throw new Error('No service or userfarestages info found');
         }
```

Comment:
We should get rid of the `isCookiesUuidMatch()` check as it has caused issues already on the matching API

Reply:
Removed.